common.py

```diff
@@ -0,0 +1,18 @@
+import os
+import subprocess as sp
+
+
+def shell_execute(*args):
+    p = sp.Popen(args, stdout=sp.PIPE, stderr=sp.PIPE)
+    p.wait()
+    out = p.stdout.read()
+    err = p.stderr.read()
+    if p.returncode == 0:
+        if type(out) == bytes:
```

Comment:
@lolski `type(err)` could be `bytes` as well then?

Reply:
under which condition would we get back bytes instead of string again, @vmax ?

Reply:
@lolski when running script with Python 3